RPA Manager release notes 2023-05/1. (#169)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[may-15-2023]]

> == May 15, 2023

Reviewer: This release should appear in Latest Releases because it's got a What's New.

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

* RPA Manager now shows a top navigation bar.

> * The *Execution* tab of from the Run Configuration is improved: 

Reviewer: ```suggestion
* The *Execution* tab for Run Configuration is improved: 
```

Reviewer: Suggest rewriting to be active voice.

> ** Option *Run in existing desktop session as logged in user* is renamed to *Run in existing Console or RDP session if specified user is signed in*.

Reviewer: ```suggestion
** The *Run in existing desktop session as logged in user* option is now *Run in existing Console or RDP session if specified user is signed in*.
```
